release_notes/374-release-notes.adoc

```diff
@@ -402,6 +403,16 @@ Release date: 13 July 2023
 
 * Provides a Python3 security update. (link:https://access.redhat.com/errata/RHSA-2023:3591[RHSA-2023:3591])
 
+[id="resolved-in-version-3746"]
+=== Resolved in version 3.74.6
+
+Release date: 2 October 2023
```

Comment:
The date could be today or tomorrow, on the downstream side we're ready to release.